libraries/chain/controller.cpp

```diff
@@ -3630,8 +3629,6 @@ struct controller_impl {
 
    template<typename BSP>
    void log_applied(controller::block_report& br, const BSP& bsp) const {
-      if (replaying) // fork_db_root_block_num not available during replay
```

Comment:
Why is this removed?

Reply:
Because it is not needed anymore. It was there before because we were logging fork_db_root_block_num and fork_db is not available during replay. However, we are no longer logging fork_db_root_block_num, so it is not needed. Also I noticed that we were not logging every 1000 blocks during replay. This restores the behavior of logging every 1000 blocks during replay.